src/app/shared/components/message/message.component.scss

```diff
@@ -0,0 +1,57 @@
+@mixin transition($args...) {
+    -webkit-transition: $args;
+    -moz-transition: $args;
```

Comment:
I think angular cover these for you

Reply:
This is interesting, can you explain it to me?